rq_dashboard/version.py

```diff
@@ -1 +1 @@
-VERSION = '0.5.2'
+VERSION = '0.5.3'
```

Comment:
Please don't bump the version in pull-requests. It's usually done in separate commit when the changes in master are stabilized.

Reply:
Sorry, didn't know! I've undone that change.